src/frontend/src/lib/actors/actors.ic.ts

```diff
@@ -42,6 +43,36 @@ export const getBackendActor = async ({
 	return backend;
 };
 
+export const getSignerActor = async ({
+	identity,
+	certified = true
+}: {
+	identity: OptionIdentity;
+	certified?: boolean;
+}): Promise<SignerActor> => {
+	// TODO: Delete this dependency once the signer has been stripped down.
+	assertNonNullish(identity, get(i18n).auth.error.no_internet_identity);
+
+	const { signer } = actors ?? { signer: undefined };
+
+	if (isNullish(signer)) {
+		const actor = await createActor<SignerActor>({
+			canisterId: SIGNER_CANISTER_ID,
+			idlFactory: certified ? idlCertifiedFactoryBackend : idlFactoryBackend,
```

Comment:
shouldn't here go the signer's idlFactories?

Reply:
Good catch!  Done.

Reply:
Done